src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -122,11 +123,14 @@ def test_edit(self):
         assert unicode(addon.name) == data['name']
         assert addon.name.id == old_name.id
 
-        assert unicode(addon.slug) == data['slug']
         assert unicode(addon.summary) == data['summary']
 
         if self.listed:
             assert [unicode(t) for t in addon.tags.all()] == sorted(self.tags)
+            assert unicode(addon.slug) == data['slug']
+        else:
+            # We randomize slugs for unlisted add-ons
+            assert len(data['slug']) == 20
```

Comment:
???  `data['slug']` should be `'test_slug'` shouldn't it?  Why would it be 20 characters long?

Reply:
eh, true, nevermind. That was indeed from an old patch and is actually failing.